synapse/rest/key/v2/remote_key_resource.py

```diff
@@ -20,17 +20,13 @@
 
 from synapse.api.errors import Codes, SynapseError
 from synapse.crypto.keyring import ServerKeyFetcher
-from synapse.http.server import (
-    DirectServeResource,
-    respond_with_json_bytes,
-    wrap_json_request_handler,
```

Comment:
`wrap_json_request_handler` seems to only be used in one place now (`AdditionalResource`)... perhaps that can be a `DirectServerJsonResource`?